bot/plotting/random_plot_generator.py

```diff
@@ -166,45 +200,58 @@ def random_plot_generator(
                 return
 
             elif choice == 1:
+
+                # generating a random experiment
+                cycle_received = experiment_generator()
+                number = random.randint(4, 100)
```

Comment:
very minor, but why not just let `experiment_generator` choose the number of repeats? Ah, I see that you want to pick a big/small number for long/short experiments.

Reply:
Yes, it was doing that before but to clean out the code I decided to shift the number out.